larq/layers_test.py

```diff
@@ -165,6 +202,19 @@ def test_layer_does_not_warn(caplog):
     assert caplog.records == []
 
 
+def test_depthwise_layer_warns(caplog):
+    lq.layers.QuantDepthwiseConv2D(5, depthwise_quantizer="ste_sign")
+    assert len(caplog.records) == 1
+    assert "depthwise_constraint" in caplog.text
+
+
+def test_depthwise_layer_does_not_warn(caplog):
```

Comment:
This is basically the same test as `test_separable_layer_does_not_warn` and `test_layer_does_not_warn`. Could you consolidate them?

Reply:
I see your point but a merged version would be harder to read and not much shorter. For unit tests, having some duplication is fine I think.

Reply:
👍 Sounds good to me